packages/@react-aria/autocomplete/src/useAutocomplete.ts

```diff
@@ -201,6 +201,18 @@ export function UNSTABLE_useAutocomplete(props: AriaAutocompleteOptions, state:
         // But what about wrapped grids where ArrowLeft and ArrowRight should navigate left/right
         clearVirtualFocus();
         break;
+      case 'Tab':
+        // Moving forward will always go into the collection which will handle Tab itself, prevent the browser's default tab behavior so both don't happen
```

Comment:
Shouldn't the collection be non-focusable because it uses virtual focus? So why would it go into the collection at all in the first place?

Reply:
We re-fire the event on the collection, so maybe more clear language would be
```suggestion
        // Tab forward will re-fire in the collection which will handle Tab itself, prevent the browser's default tab behavior so both don't happen
```
But this is the line we want to actually handle the Tab https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/selection/src/useSelectableCollection.ts#L288

Reply:
oh, maybe `useSelectableCollection` should just turn off its tab handler if `shouldUseVirtualFocus` is true?

Reply:
oh btw, looks like Daniel also already found this in his PR: https://github.com/adobe/react-spectrum/pull/7561/files#diff-438043cb2e3951363a3350ad94e45c05c063a3cea6dcb4c360e8a01848b2fcc5R193

Reply:
o, nice, looks like it passes the tests I wrote as well
want to take these tests into your PR? otherwise I've pushed the change he made here as well